exercises/concept/log-parser/.meta/design.md

```diff
@@ -0,0 +1,30 @@
+# Design
+
+We assume that the student already knows basic regular expressions. The goal of this exercise is to teach them Elixir-specific details only.
+
+## Learning objectives
+
+- Know about the Regex module
+- Know about the `=~` operator
+- Know that some String functions accept regular expressions, e.g. match?, replace, split.
+- Know about modifiers (e.g. unicode, case-insensitive)
+- Know how to get a value from a captured group
+- Know that sigils can be used with different delimiters
+
+## Out of scope
+
+- Teaching the syntax of regular expressions
+- Compiling Regular expressions with variable content
```

Comment:
I wanted to add some step that would require an interpolated regex with escaping a variable, but I couldn't come up with anything thematic that wouldn't be better done with `String.starts_with?` :/